node/package/repo/install.sls

```diff
@@ -4,7 +4,7 @@
 {#- Get the `tplroot` from `tpldir` #}
 {%- set tplroot = tpldir.split('/')[0] %}
 {%- from tplroot ~ "/map.jinja" import node with context %}
-{%- from tplroot ~ "/macros.jinja" import format_kwargs with context %}
+{%- from tplroot ~ "/macros/macros.jinja" import format_kwargs with context %}
```

Comment:
This PR's changeset shows `node/macros.jinja → node/files/macros.jinja`, so `"/macros/macros.jinja"` would have to be fixed accordingly.

Reply:
Thanks, not sure why I missed that. Sloppy.